src/_data/starters/11ty-zuixjs.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/zuixjs/zuix-web-starter",
+	"name": "zuix-web-starter",
+	"description": "Mobile first and component based web starter template (PWA).",
+	"author": "genemars",
```

Comment:
@genemars Is this your Twitter account?

Reply:
No, that's my github account. Updated with **genejams**, that's my twitter.